.gitignore

```diff
@@ -23,5 +23,3 @@ yarn-debug.log*
 yarn-error.log*
 
 .idea
-
-package-lock.json
```

Comment:
Why we need this file in repository? This file is auto generated every time when you execute npm install.

Reply:
that file needed to resolve some dependencies properly (some deps still not support last react version)v
working on that

Reply:
Please try  run  [```npm install --legacy-peer-deps```](https://github.com/kpi-ua/rozklad-kpi/blob/master/.dockerfile#L8).

Using a cached or manually edited package-lock.json is not a good approach.